core/trino-parser/src/main/java/io/trino/sql/ReservedIdentifiers.java

```diff
@@ -71,28 +72,22 @@ private static void validateDocs(Path path)
         System.out.println("Validating " + path);
         List<String> lines = Files.readAllLines(path);
 
-        if (lines.stream().filter(s -> s.startsWith(TABLE_PREFIX)).count() != 3) {
+        if (lines.stream().filter(s -> s.startsWith(TABLE_START)).count() != 1) {
             throw new RuntimeException("Failed to find exactly one table");
         }
 
         Iterator<String> iterator = lines.iterator();
 
-        // find table and skip header
-        while (!iterator.next().startsWith(TABLE_PREFIX)) {
+        // find start of list in table
+        while (!iterator.next().startsWith(TABLE_START)) {
             // skip
         }
-        if (iterator.next().startsWith(TABLE_PREFIX)) {
-            throw new RuntimeException("Expected to find a header line");
-        }
-        if (!iterator.next().startsWith(TABLE_PREFIX)) {
-            throw new RuntimeException("Found multiple header lines");
-        }
 
         Set<String> reserved = reservedIdentifiers();
         Set<String> found = new HashSet<>();
         while (true) {
             String line = iterator.next();
-            if (line.startsWith(TABLE_PREFIX)) {
+            if (line.startsWith(TABLE_END)) {
```

Comment:
There's no need to put dummy line `% END`. We can break the loop when the line doesn't start with `|`.

Reply:
Well .. I thought its better to be explicit rather than implicit .. but fine .. I will change.